### DISCUSSION TITLE
Feature Request: Row Links.

### DISCUSSION THREAD

**Discussion by EvilLaughInc** [grafana/grafana#53271] [General]
Hello, 

I am working on two dashboards. One is a template dashboard that can be thrown onto any given client site and instantly report metrics.  The other dashboard is a aggregated client site monitoring dash. I have new rows being auto generated by the client site, but would really like to be able to push a button or link in the row and be directed to the corresponding dash. While there is a dashboard link panel, and the ability to add links to panels, they tend to be large (when you can see the number of sites you want to see), too small (to see full client site names), or static, which would require manually changing links when the destination changes. 

I would really appreciate it if there was a way to link rows to dashboards on the same Grafana deployment. Ideally it would have options for both auto generated AND manual addition. It could very easily be another pop up icon directly next to the row name, like the settings or delete options. 

I'm really after a one-click solution to redirect to desired dashboards that can be set rapidly and automatically following a determined syntax. Thank you.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
